src/types/onyx/Report.ts

```diff
@@ -183,6 +183,11 @@ type Report = OnyxCommon.OnyxValueWithOfflineFeedback<
         pendingChatMembers?: PendingChatMember[];
 
         /** If the report contains reportFields, save the field id and its value */
+        reportFields?: Record<string, PolicyReportField>;
```

Comment:
Seems like this is added on the same place we used the `fieldList`, why did we change this? is the fieldList still used?